### PR TITLE
Support to Hop into tenants

### PR DESCRIPTION
**Pull request by dvaldivia**
This adds the capability to browse tenants as an admin from the Tenant Details screen

Screen Shot 2021-07-20 at 2 15 11 PM
Screen Shot 2021-07-20 at 2 15 08 PM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+              }}
+            >
+              <ExitToAppIcon />
+            </IconButton>
+          </React.Fragment>
+        }
+      />
+      <div className={classes.divContainer}>
+        {loading && (
+          <div style={{ width: "100px", margin: "auto", marginTop: "80px" }}>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx:114]: Please not hardcode styles, add them as a class in createStyles

File: portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    breadcrumLink: {
+      textDecoration: "none",
+      color: "black",
+    },
+    iframeStyle: {
+      border: "0px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx:39]: Please change this to be integer
```suggestion
      border: 0,
```

File: portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+      position: "absolute",
+      height: "calc(100vh - 77px)",
+      width: "100%",
+    },
+    divContainer: {
+      position: "absolute",
+      left: "0px",
+      top: "77px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx:47]: Please change this to integer values
```suggestion
      left: 0,
      top: 77,
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx:114]: done

**Review by bexsoft — DISMISSED** [review]
LGTM

**Review by Alevsk — DISMISSED** [review]

**Review by bexsoft — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Pull request merged**